resources/sql/application-queries.sql

```diff
@@ -835,3 +835,18 @@ JOIN latest_forms AS lf ON lf.key = f.key
 WHERE a.person_oid = :person_oid
   AND (:query_type = 'ALL' OR lf.organization_oid IN (:authorized_organization_oids))
 ORDER BY a.created_time DESC;
+
+--name: yesql-tilastokeskus-applications
+SELECT
+  haku AS haku_oid,
+  key AS hakemus_oid,
+  person_oid hekilo_oid,
+  hakukohde AS hakukohde_oids
+FROM latest_applications
+  JOIN application_reviews ON application_key = key
+WHERE person_oid IS NOT NULL
+  AND haku IS NOT NULL
+  AND hakukohde != '{}'
```

Comment:
Tuon `haku IS NOT NULL` pitäis taata, että hakukohteita on. Jos ei, niin sit meil on ongelma.